libr/bin/format/elf/elf.c

```diff
@@ -75,6 +75,47 @@ static inline int UTX_MUL(ut64 *r, ut64 a, ut64 b) {
 
 #define round_up(a) ((((a) + (4) - (1)) / (4)) * (4))
 
+#define EF_MIPS_ABI_O32		0x00001000  /* O32 ABI.  */
+#define EF_MIPS_ABI_O64		0x00002000  /* O32 extended for 64 bit.  */
+#define EF_MIPS_ABI			0x0000f000
+
+/*
+ * Return true if HDR identifies an MIPS n64 ELF binary.
```

Comment:
this comment is wrong. the function returns true if its 64 bit. nothing related to mips. can you remove the misleading comment?